jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -1648,7 +1729,21 @@ def dot_product_attention(
     bias: Bias to be added to logits with a shape of BNTS.
     mask: Mask used to filter out logits with a shape of BNTS.
     q_seqlen: Non padded sequence length of Queries with a shape of B.
+      If q_offsets is set, q_seqlen should have shape [B,M] where M is the
+      maximum number of segments per batch. Fill the invalid entries with -1.
```

Comment:
"invalid" looks a bit out of place here. Would it be more clear to say that -1 is the padding value?

Please update other occurrences if you update this one.